main.go

```diff
@@ -69,7 +69,7 @@ func main() {
 	flag.Parse()
 
 	if *flagVersion {
-		fmt.Println(version)
+		fmt.Fprintf(flag.CommandLine.Output(), "%s(%s)\n", version, revision)
```

Comment:
Can we only output `dev` for non-released version, rather than `dev(dev)`?

Reply:
I agree I'll change 😄

Reply:
If `revision` is not specified, only `dev` is displayed.

- before

```
$ go build -o bin/aztfy              
$ bin/aztfy -v
dev(dev)
```

- after

```
$ go build -o bin/aztfy              
$ bin/aztfy -v
dev
```

Reply:
> Seems there will be two outputs when the revision is specified?

I'm bad 🙈  fix it.

I checked below.

- no version, no revision

```
$ CGO_ENABLED=0 go build -o bin/aztfy
$ bin/aztfy -v
dev
```

- version, no revision

```
$ CGO_ENABLED=0 go build -trimpath -ldflags="-s -w -X 'main.version=v0.1.0'" -o bin/aztfy
$ bin/aztfy -v                                                                           
v0.1.0
```

- version, revision

```
$ CGO_ENABLED=0 go build -trimpath -ldflags="-s -w -X 'main.version=v0.1.0' -X 'main.revision=xxxxx'" -o bin/aztfy
$ bin/aztfy -v                                                                                                    
v0.1.0(xxxxx)
```